### ISSUE TITLE
seq2seq推理部分有点问题

### ISSUE THREAD

**Issue by wdwcn** [zhaoyingjun/chatbot#52]
execute.py下的decode_line（）函数会把中文词转为二进制表示，所以在词字典中是找不到的，token_ids一直是[3,3,3,3…]

**Comment by zhaoyingjun** [comment]
是你的文件编码的问题

**Comment by wdwcn** [comment]
有点记不清当时都改了哪些地方，我把大佬的代码注释了一部分，绕过了转二进制部分。其他地方有没有修改不是太确定了。
def decode_line(sess, model, enc_vocab, rev_dec_vocab, sentence):
    # Get token-ids for the input sentence.
    #token_ids = prepareData.sentence_to_token_ids(tf.compat.as_bytes(sentence), enc_vocab)
    #token_ids = prepareData.sentence_to_token_ids(sentence, enc_vocab)
    def sentence_to_token_ids(sentence, enc_vocab):
        token_ids=[]
        for word in re.split(r'\s+',sentence):
            token_ids.append(enc_vocab.get(word,3))
        return(token_ids)
    token_ids=sentence_to_token_ids(sentence,enc_vocab)

    # Which bucket does it belong to?
    bucket_id = min([b for b in xrange(len(_buckets)) if _buckets[b][0] > len(token_ids)])

    # Get a 1-element batch to feed the sentence to the model.
    encoder_inputs, decoder_inputs, target_weights = model.get_batch({bucket_id: [(token_ids, [])]}, bucket_id)

    # Get output logits for the sentence.
    _, _, output_logits = model.step(sess, encoder_inputs, decoder_inputs, target_weights, bucket_id, True)

    # This is a greedy decoder - outputs are just argmaxes of output_logits.
    outputs = [int(np.argmax(logit, axis=1)) for logit in output_logits]

    # If there is an EOS symbol in outputs, cut them at that point.
    if prepareData.EOS_ID in outputs:
        outputs = outputs[:outputs.index(prepareData.EOS_ID)]

    return " ".join([tf.compat.as_str(rev_dec_vocab[output]) for output in outputs])
